waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -40,29 +42,33 @@ proc insertRow(): string =
  """INSERT INTO messages (id, storedAt, contentTopic, payload, pubsubTopic,
   version, timestamp) VALUES ($1, $2, $3, $4, $5, $6, $7);"""
 
-const MaxNumConns = 5 #TODO: we may need to set that from app args (maybe?)
+const MaxNumConns = 50 #TODO: we may need to set that from app args (maybe?)
```

Comment:
Random question. In most systems, there's significantly more read operations than write ones.

Does it makes sense to instead of dividing the max connections equally, to give a higher maximum to read ones and a lower one for write? Would it help in anything?

Reply:
Good point! :heart: 
However, in the tests made so far, it never reached the parallel 50 connection so for now I'd leave it in with that naïve separation until we find an empirical reason for that.